rcl_lifecycle/src/com_interface.c

```diff
@@ -89,15 +88,28 @@ rcl_lifecycle_com_interface_init(rcl_lifecycle_com_interface_t * com_interface,
   }
 
   const char * node_name = rcl_node_get_name(node_handle);
+  size_t node_name_length = strlen(node_name);
+
+  // Build topic, topic suffix hardcoded for now
+  // and limited in length of 255
+  // e.g. lc_talker__transition_event
+  if ( ((node_name_length + 2 + strlen(pub_transition_event_suffix)) > 255) ||
+    ((node_name_length + 2 + strlen(srv_get_state_suffix)) > 255) ||
+    ((node_name_length + 2 + strlen(srv_change_state_suffix)) > 255) ||
+    ((node_name_length + 2 + strlen(srv_get_available_states_suffix)) > 255) ||
+    ((node_name_length + 2 + strlen(srv_get_available_transitions_suffix)) > 255))
```

Comment:
Sorry I wasn't necessarily suggesting that. You'll have to come back and touch these when all support for namespaces is done. I didn't realize it was already being checked like this.